indico/modules/receipts/controllers/templates.py

```diff
@@ -224,3 +226,18 @@ def _process(self):
         new_tpl.log(new_tpl.log_realm, LogKind.positive, 'Templates', f'Document template "{new_tpl.title}" created',
                     user=session.user, data={'Cloned from': self.template.title})
         return jsonify(ReceiptTemplateDBSchema().dump(new_tpl))
+
+
+class RHGetDefaultTemplate(RHProtected):
+    """Get a template from a default template."""
+
+    def _process(self):
+        name = request.view_args['template_name']
```

Comment:
I'd prefer using a whitelist for this (e.g. using `templates_path.glob('*.yaml')`) instead of using it directly in a path